internal/service/rds/cluster_test.go

```diff
@@ -6421,9 +6423,14 @@ resource "aws_security_group_rule" "test" {
   security_group_id = aws_security_group.test.id
 }
 
+data "aws_secretsmanager_random_password" "test" {
+  password_length     = 20
+  exclude_punctuation = true
+}
+
 resource "aws_directory_service_directory" "directory" {
```

Comment:
This resource does not support `password_wo` yet. When it does, this should be changed to an ephemeral.